.github/workflows/test-php.yml

```diff
@@ -59,6 +64,17 @@ jobs:
                   node --version
                   npm --version
                   timedatectl
+            - name: Add PHP8 Compatibility.
```

Comment:
Same as the way woo core does it -> https://github.com/woocommerce/woocommerce/blob/trunk/.github/workflows/pr-unit-tests.yml#L57